test/browser/full_page_refresh_test.rb

```diff
@@ -51,4 +51,22 @@ class FullPageRefreshTest < ActionDispatch::IntegrationTest
     page.evaluate_script('window.history.back()')
     assert_equal "Sample Turbograft Application", page.title
   end
+
+  test "tg-static preserves client-side state of innards on partial refresh, and replaces contents on full refresh" do
+    page.fill_in 'badgeinput', :with => 'tg-static innards'
+    click_link "Perform a full page refresh"
+    assert_equal "tg-static innards", find_field("badgeinput").value
+    click_link "Perform a partial page refresh and refresh the navigation section"
+    assert_equal "", find_field("badgeinput").value
```

Comment:
Is this test correct?
According to line `57` and `58` you do a fullpage refresh and the value is kept. 
On `59` and `60` you do a partial and the value is replaced.

The test title is saying the opposite.